packages/ui/src/components/tables/TableContent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import '@ferlab/style/components/tables/TableContent.scss'
+
+export type TableContentProps = {
+    className?: string;
+    children?: React.ReactNode;
+};
+
+const TableContent: React.FC<TableContentProps> = ({
```

Comment:
Not sure about the name of the component though... ScrollableTable was saying more on what was the component doing